src/pages/en/guides/styling.md

```diff
@@ -161,7 +165,32 @@ Use `<style lang="less">` in `.astro` files.
 - **Vue**: `<style lang="scss">`
 - **Svelte**: `<style lang="scss">`
 
-Additionally, PostCSS is supported as an [integration](/en/guides/integrations-guide/).
+## PostCSS
+
+Astro comes with PostCSS included as part of [Vite](https://vitejs.dev/guide/features.html#postcss). To configure PostCSS for your project. Make a `postcss.config.js` in the root of the project directory. Import plugins from [npm](https://www.npmjs.com/search?q=postcss). Using `require()` we apply additional plugins to the configuration file.    
+
+```js
+//./postcss.config.js
+// Import a PostCSS plugin (https://www.postcss.parts/)
+const postcssPresetEnv = require('postcss-preset-env');
+
+module.export={
+  // Plugins as an Array
+  plugins:[
+   postcssPresetEnv({
+     //Plugin options
+   }),
+  autoprefixer({
+    ...options
+  }),
+  ],
+  // Plugins as an object
+  plugins:{
+    postcssPresetEnv:{...options},
+    autoprefixer:{...options},
+  }
+}
```

Comment:
What do you think about trimming this example down a bit for clarity?

Maybe something like:

```suggestion
// ./postcss.config.js

module.exports = {
  plugins: [
    require('postcss-preset-env'),
    require('autoprefixer'),
  ],
};
```

I’m not sure we need to show every possible configuration style, just something to get folks started.

Reply:
Shorter is often good for cognitive load! Do we feel the need to show what it looks like with options?

Reply:
I don’t think either of these plugins have commonly used options. PostCSS’s own README literally doesn’t show how to use options 😆